lib/Quantization/Quantization.cpp

```diff
@@ -121,7 +133,9 @@ class FunctionQuantizer : public FunctionConverter {
   ///      the other must be a quantized type.
   Node *createConversion(NodeValue &val, TypeRef destTy) override {
     if (destTy->isQuantizedType()) {
-      assert(destTy->getElementType() == ElemKind::Int8QTy && "");
+      assert((destTy->getElementType() == ElemKind::Int8QTy ||
+              destTy->getElementType() == ElemKind::Int32QTy) &&
+             "");
```

Comment:
i'd just remove this "" it does not bring any value, or provide better error message.